### ISSUE TITLE
Outrun end-race progress map is shifted partly off-screen

### ISSUE THREAD

**Issue by Atariboy1982** [agg23/openfpga-pcengine#27]
### Version (or build number)

1.0.0

### Steps to reproduce

It happens every single time that the map screen is displayed. No special steps are necessary to reach that point in the game since the end race map that shows your progress will appear after the timer runs down even if you never press the accelerator button at the starting line. 

### Expected Behavior

I'm not a TurboGrafx-16/PC Engine owner, but I would expect to see it properly displayed. YouTube uploads from real hardware show it looking as I'd expect it to, such as this screen capture that I just took from a random longplay video.
![example](https://github.com/user-attachments/assets/a4bff02d-d4fd-4d4f-bdcb-df8426b30944)

### Actual Behavior

Map screen is significantly shifted off-screen every time this screen is displayed. Also note that I don't own a dock at this time, so I can only attest to this happening in handheld mode. First example is what the screenshot taken by the Pocket looks like, which isn't accurate to what's actually displayed on screen. Second is a quick mockup to approximate what it looks like in-game on the Pocket's screen.

![20240124_083702](https://github.com/user-attachments/assets/6d211114-a95c-4e20-aa5f-71876f139065)![ex](https://github.com/user-attachments/assets/6d3b43de-5830-4d1e-84bd-9407f737ce2e)

**Comment by MP2E** [comment]
Good catch, this appears to be a bug upstream with the MiSTer Turbografx-16 core. Something is definitely off with the results screen, it’s cut off even on my CRT, and I compared with my real Coregrafx console on the same CRT and the results screen looks normal there.

**Comment by Atariboy1982** [comment]
Just to add something that I noticed after I posted but didn't have a chance to double check on until just now, the icon representing your Ferrari as your progress and path are drawn on the map is MIA on the map screen as well. 

I think I had noticed the missing Ferrari months ago (I've known about the bugged map screen for ages but I'm lazy), but I just assumed it was trimmed back for one reason or another with this port and wasn't supposed to be there and thus wasn't a bug. But it's quite clearly evident in the screen grab that I made off of a YouTube upload.
